src/css/components/section.scss

```diff
@@ -120,10 +120,14 @@ $chevron-size: 4.865rem;
   a:active,
   a:hover,
   a:visited {
-    color: $color-red;
-    text-decoration: underline;
+    // color: $color-red;
+    // text-decoration: underline;
+    color: $color-blue;
+    text-decoration: none;
+    border-bottom: 1px solid $color-blue;
   }
-
+  // these seem to just override the definitions immediately above it
+  // why keep both?
```

Comment:
this is is so they don't have to copy the text-decoration and border-bottom part to both sets.
the second set is just to override the color.